### ISSUE TITLE
zoralabTABLE Has Header Rendering UI Bug

### ISSUE THREAD

**Issue by corygalyna** [ZORALab/Hestia#385]
# Description
```
Please provide a short description of what you have encountered below.
```
Rendered table UI has colossal bug when the headers are wrapped (e.g. `code` and `pre`).

# Expected Behavior
```
Please specify the expected behavior of your usage below.
```

![expected-table-2023-06-28-12-38-34](https://github.com/ZORALab/Hestia/assets/124406765/2ff8c647-c348-4793-9ae5-159da700d753)

# Current Behavior
```
Please specify the current behavior of software below.
```

![bad-table-2023-06-28-12-35-11](https://github.com/ZORALab/Hestia/assets/124406765/9d392486-d046-4ecf-91e4-bd5012f72b7f)

# Steps to Reproduce (COMPULSORY)
```
Please specify all the steps required to re-produce the problem.
```
1. Visit: https://hestia.zoralab.com/en/specs/hestiagui/zoralabcore/

# Attachments
```
Please drag and drop the necessary data files (e.g. screenshot, logs, etc)
below.
```

**Comment by hollowaykeanho** [comment]
Fixed in a581e26c74ae92e5ebb5b07d7068753142abc066

Table now appear as:

![table-fixed](https://github.com/ZORALab/Hestia/assets/7259572/b5644db1-ec22-47f6-8efa-4d7bb0632373)

**Comment by hollowaykeanho** [comment]
Released in v1.2.1.
